tests/unit/contact-form.test.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env jasmine */
+describe('Restaurant Week', () => {
+
+	beforeEach(() => {
+		
+	});
+
+	describe('Contact Form', () => {
+		it('should load the Baltcogo app', () => {
+			expect(false).toBe(true);
```

Comment:
You can remove this since it's pointless at the moment.

Reply:
done